### PR TITLE
[FIX] Move translations in i18n_extra

### PR DESCRIPTION
**Pull request by Ehtaga**

### CODE REVIEW

File: res_country_state_translations_canada/i18n_extra/fr.po [hunk 1]
```diff
@@ -0,0 +1,48 @@
+# Translation of Odoo Server.
```
Comment (joaoalf) [inline res_country_state_translations_canada/i18n_extra/fr.po:1]: @Ehtaga Where's Quebec ?

File: res_country_state_translations_us/i18n_extra/fr.po [hunk 2]
```diff
@@ -0,0 +1,138 @@
+# Translation of Odoo Server.
```
Comment (joaoalf) [inline res_country_state_translations_us/i18n_extra/fr.po:1]: @Ehtaga Also there are some states missing.

**Review by joaoalf — CHANGES_REQUESTED** [review]
Why are we using i18n_extra?

**Comment by Ehtaga** [comment]
@joaoalf I'm using i18n_extra because the OCA Travis bot automatically remove the translations in i18n folder, as they don't belong to this specific module (we're translating `base` data)

See https://github.com/OCA/l10n-canada/commit/c856d95921307ac7fd681d86b9e898e9969f526a

File: res_country_state_translations_canada/i18n_extra/fr.po (hunk 1 above)
Comment (Ehtaga) [reply res_country_state_translations_canada/i18n_extra/fr.po:1]: @joaoalf Lost in the separation process! It's back!

File: res_country_state_translations_us/i18n_extra/fr.po (hunk 2 above)
Comment (Ehtaga) [reply res_country_state_translations_us/i18n_extra/fr.po:1]: @joaoalf I did not add the states who are the same in English and in French. Only those with a translation are there.

**Review by rimbendhaou — COMMENTED** [review]
@Ehtaga sometimes  Rhode Island is translated to Île-de-Rhodes in french

**Comment by joaoalf** [comment]
@Ehtaga It looks like the ca module has no icon.

**Comment by Ehtaga** [comment]
@joaoalf Both had an icon, but it was the canadian flag, so I put the US one in the us module.

@rimbendhaou It's not the official translation so we should keep Rhode Island.

**Review by joaoalf — APPROVED** [review]

**Review by rimbendhaou — APPROVED** [review]
ok

**Pull request merged**